python/private/config_settings.bzl

```diff
@@ -209,3 +209,33 @@ _current_config = rule(
         "_template": attr.string(default = _DEBUG_ENV_MESSAGE_TEMPLATE),
     },
 )
+
+def is_python_version_at_least(name, **kwargs):
+    flag_name = "_{}_flag".format(name)
+    native.config_setting(
+        name = name,
+        flag_values = {
+            flag_name: "yes",
+        },
+    )
+    _python_version_at_least(
+        name = flag_name,
+        visibility = ["//visibility:private"],
+        **kwargs
+    )
+
+def _python_version_at_least_impl(ctx):
+    at_least = tuple(ctx.attr.at_least.split("."))
+    current = tuple(
+        ctx.attr._major_minor[config_common.FeatureFlagInfo].value.split("."),
+    )
+    value = "yes" if current >= at_least else "no"
```

Comment:
This version matching is a little bit brittle. Might be better to use `evaluate` from `pep508_evaluate` where we use `"python_version >= {}".format(ctx.attr.at_least)` for the marker and then evaluate by passing in the `env`.

I think it is fine for now to have your implementation, but at some point it may be nicer to use the standard evaluation.